packages/design-tokens-typescript/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@gwyneplaine/design-tokens-typescript",
```

Comment:
Are we adding a ts format? Maybe this should live in a different PR w/ a Jira ticket